next-env.d.ts

```diff
@@ -9,3 +9,10 @@ declare module '*.svg' {
   const src: string;
   export default src;
 }
+
+type RuntimeConfig = {
```

Comment:
This might be actually happening only on Next 11 and up. I believe this file was documented to persist modifications and this is no longer the case with new versions.

Reply:
Good catch. The documentation says not to change this file. I moved all our types to `additional.d.ts`